scripts/run.sh

```diff
@@ -142,13 +142,24 @@ function dockerPrune() {
         --filter="label!=com.bitwarden.project=setup"
 }
 
+function preventLetsEncryptRandomSleepOnRenew() {
+    if [ -f "${OUTPUT_DIR}/config.yml" ]
+    then
+        grep -E "^lets_encrypt_no_random_sleep_on_renew: true" "${OUTPUT_DIR}/config.yml" 2>&1 >/dev/null
+        if [ "$?" -eq 0 ]
```

Comment:
No need to run grep separately from test ("["). You can just

```
if grep -E "^lets_encrypt"...; then
```

Also, "-q" stops grep stdout.

Reply:
Thanks for the review! I've updated with your suggestions.